solidity/random-beacon/contracts/libraries/Relay.sol

```diff
@@ -192,6 +192,29 @@ library Relay {
         return (inactiveMembers, slashingAmount);
     }
 
+    /// @notice Validates if the reporting of an unauthorized group signing can
+    ///         be performed. The signature verification is done by BLS lib.
+    /// @param signedMsgSender Signature of the sender's address as a message.
+    /// @param group Group that is being verified against private key leakage.
+    /* solhint-disable no-unused-vars */
+    function validateUnauthorizedSigning(
+        Data storage self,
+        bytes memory signedMsgSender,
+        Groups.Group memory group
+    ) internal {
+        /* solhint-enable no-unused-vars */
+        require(!group.terminated, "Group cannot be terminated");
```

Comment:
We are terminating group in `RandomBeacon` so maybe we should have this check there? Then, we do not have to pass the entire `Group` but just `group.groupPubKey`.

Reply:
Yes, moved back to RandomBeacon contract e8f169d